features/unattached_commands.feature

```diff
@@ -218,3 +218,70 @@ Feature: Command behaviour when unattached
           | kinetic | disable  |
           | jammy   | enable   |
           | jammy   | disable  |
+
+    @wip
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Check for newer versions of the client in an ubuntu machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        Then I verify that no files exist matching `/tmp/ubuntu-advantage/candidate-version`
+        When I run `pro status` as non-root
+        Then I will see the following on stderr
+        """
+        """
+        And I verify that files exist matching `/tmp/ubuntu-advantage/candidate-version`
+        # We forge a candidate to see results
+        When I delete the file `/tmp/ubuntu-advantage/candidate-version`
+        And I create the file `/tmp/ubuntu-advantage/candidate-version` with the following
+        """
+        99.9.9
+        """
+        And I run `chmod 777 /tmp/ubuntu-advantage/candidate-version` with sudo
+        And I run `pro status` as non-root
+        Then I will see the following on stderr
+        """
+        *Warning*: A new version is available: 99.9.9.
```

Comment:
@orndorffgrant @CalvoM @renanrodrigo should we move this message to the beginning of the output instead ?

Reply:
I think it is less invasive to have it at the bottom, but can be convinced otherwise.

Reply:
I think it is less jarring and more likely to be seen at the bottom.

I've been thinking of pip and npm as we develop this, which both do their update notice at the bottom IIRC.

I just verified with pip and accidentally found out that there actually is a pip package named "nonsense" :shrug: 
```
> pip install nonsense
Collecting nonsense
  Downloading nonsense-0.1.0.tar.gz (1.4 kB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: nonsense
  Building wheel for nonsense (setup.py) ... done
  Created wheel for nonsense: filename=nonsense-0.1.0-py3-none-any.whl size=2026 sha256=ad5fddb17be4cfe4459df962013703421b6bc7fddd0080c113654ac35a36982f
  Stored in directory: /home/grant/.cache/pip/wheels/8a/f3/b9/2853d7947af0ac3f214067ffd60ba431349f292eabbada2562
Successfully built nonsense
Installing collected packages: nonsense
Successfully installed nonsense-0.1.0

[notice] A new release of pip available: 22.1.2 -> 22.2.2
[notice] To update, run: pip install --upgrade pip
```